app/controllers/katello/api/v2/errata_controller.rb

```diff
@@ -20,8 +20,12 @@ def available_for_content_view_filter(filter, collection)
       collection = filter_by_content_view(filter, collection)
       ids = Katello::ContentViewErratumFilterRule.where(:content_view_filter_id => filter.id).pluck("errata_id")
       collection = collection.where("errata_id not in (?)", ids) unless ids.empty?
-      collection = collection.where('issued  >= ?', params[:start_date]) if params[:start_date]
-      collection = collection.where('issued  <= ?', params[:end_date]) if params[:end_date]
+      date_type = "issued"
+      if params[:use_issued_date].present? && params[:use_issued_date] == false
```

Comment:
If this is a usable parameter for index calls I think it should be documented in the API params.